examples/image-to-text/run_pipeline.py

```diff
@@ -402,6 +407,21 @@ def preprocess(self, image, prompt=None, timeout=None):
         f"time = {(end - start) * 1000 / args.n_iterations}ms, Throughput (including tokenization) = {throughput} tokens/second"
     )
 
+    stats = ""
+    stats = stats + f"\nThroughput (including tokenization) = {throughput} tokens/second"
+    stats = stats + f"\nNumber of HPU graphs                = {count_hpu_graphs()}"
+    separator = "-" * len(stats)
```

Comment:
@jaygala223  why don't we use https://docs.habana.ai/en/latest/PyTorch/Reference/Python_Packages.html?highlight=memory%20stat#metric-apis?

Reply:
Hi @libinta, thanks for the review. I have not used this before. I mostly took reference from https://github.com/huggingface/optimum-habana/blob/231e923c83a589a029c48d5a7b2f4a2d14a9732e/examples/text-generation/run_generation.py#L805
![image](https://github.com/user-attachments/assets/65b1e004-69e0-458f-9b83-a4167d4c4726)